Source/UI/NewUI/CommonUI/View/TextFieldView.swift

```diff
@@ -26,7 +26,15 @@ class TextFieldView: UIView {
         view.backgroundColor = .white
         return view
     }()
-    
+
+    private(set) lazy var isOptionalLabel: UILabel? = {
+        let view = UILabel()
+        view.numberOfLines = 1
```

Comment:
This property is normally 1 by default which usually causes a lot of issues. Might want to set it to 0 instead, as we don't know how the text might scale in different localisations & device/font sizes.